std-bits/base/src/main/java/org/enso/base/enso_cloud/logging/LogJob.java

```diff
@@ -0,0 +1,30 @@
+package org.enso.base.enso_cloud.logging;
+
+import java.util.concurrent.CompletableFuture;
+
+public final class LogJob {
```

Comment:
Why this class is exported? Make package private.

Reply:
Done in 039b27949d88dee15ef785f348fff2c4fe329043